.github/release-drafter.yml

```diff
@@ -1,24 +1,31 @@
 # Config for https://github.com/release-drafter/release-drafter
-name-template: 'Akka Serverless Java SDK $NEXT_PATCH_VERSION'
+name-template: 'Akka Serverless Java and Scala SDKs $NEXT_PATCH_VERSION'
 tag-template: 'v$NEXT_PATCH_VERSION'
 categories:
   - title: '🚀 Features'
     labels:
       - 'Feature'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+      - 'fix'
```

Comment:
I've been looking at this again today, as well.
Having "Feature" and "Bug fixes" as separate categories might not be too helpful. The PRs will be duplicated in the listing for e.g. the "Java SDK" category. Within those categories the prefix will give some useful grouping. 
So labelling by file location might be enough for decent release notes?

Reply:
That is true and I agree. Would have been nice to have 

* Java SDK Features
* Java SDK Bug Fixes
* Scala SDK Features
* Scala SDK Bug Fixes
* Other
  - all chore and other misc stuff goes here and are not duplicated into the above categeries 

Then it might be better to only use convention commit title for the labeling and not the file locations.

Reply:
Then we also have a good reason to use of the conventional commit comments. They will drive what the release notes look like. Touched files is too blunt in my opinion. A change is often most important for some area, such as a feature in javasdk, but also touches some other things for small cleanup or consistency. The person writing that commit knows better best.